bitnami/prometheus-operator/templates/prometheus/serviceaccount.yaml

```
@@ -4,5 +4,8 @@ kind: ServiceAccount
 metadata:
   name: {{ template "prometheus-operator.prometheus.serviceAccountName" . }}
   labels: {{- include "prometheus-operator.prometheus.labels" . | nindent 4 }}
+  {{- with .Values.prometheus.serviceAccount.annotations }}
+  annotations: {{- toYaml . | nindent 4 }}
+  {{- end }}
```

Please use this approach instead that accepts templates in the annotations:
```suggestion
  {{- if .Values.prometheus.serviceAccount.annotations }}
  {{- include "prometheus-operator.tplValue" (dict "value" .Values.prometheus.serviceAccount.annotations "context" $) | nindent 4 }}
  {{- end }}
```
```